Mlem/Extensions/Navigation getter.swift

```diff
@@ -9,11 +9,11 @@ import Foundation
 import SwiftUI
 
 private struct NavigationPathGetter: EnvironmentKey {
-    static let defaultValue: Binding<NavigationPath>? = nil
+    static let defaultValue: Binding<NavigationPath> = .constant(NavigationPath())
```

Comment:
This question came up yesterday, is there any way we can attach the current community to this? that way as you navigate, you will always have the most contextual CommunityAPI for the view?

Just a thought, not a must fix.